### ISSUE TITLE
Container fluid is not respected in backend

### ISSUE THREAD

**Issue by nanu-c** [liip/bootstrap-blocks-wordpress-plugin#69]
There is no possibility for the gutenberg editor to distinguish between container and container fluid. That would be nice to reduce confusion for non tech-editor of the page.

**Comment by tschortsch** [comment]
You mean there is no way to visually distinguish between a fluid and a non-fluid container in the editor right? Styling the editor is always a bit tricky since the core Markup and Styles change with nearly every release of Gutenberg/WordPress. That's why we tried to avoid styling modifications as much as possible. But you're right it would definitely make sense to show a visual hint that the current container is fluid or not.

**Comment by nanu-c** [comment]
Yeah i mean that :). But I would say, setting an extra class in gutenberg for container fluid is enough for me, because if i really want to give a visual hint, I can add some styles to the theme.
